### ISSUE TITLE
SafeA9LHinstaller booted to a bricked unit

### ISSUE THREAD

**Issue by SG987** [AuroraWright/SafeA9LHInstaller#22]
As per the instructions found this morning on Plailects' guide (the newer otpless process), my N3DS unit on 9.6 firmware was successfully downgraded to 9.2 FW and successfully generated a NANDmin back up. After multiple checks of the various latest-version files being in the correct place on the sd card, SafeA9LHinstaller was launched. The guide prepared me for the possibility of a glitched screen but the screens upon launch were black (no signals, no backlight, no indication of activity). Rather than press select (as the guide advises if SafeA9LHinstaller launches to a glitched screen) I waited a while and powered down the system so I could try again and hopefully launch SafeA9LHinstaller again, more successfully. Unfortunately, the system would not boot again - not with the sd card or out or inserted during the boot process, not with all the sd cards contents being re-copied over to the correct locations, not after trying out every suggestion in the troubleshooting part of the guide. Recovery mode does not work either. The blue power light functions as normal, i.e. it turns on immediately upon being pressed and takes about 10 seconds or so to turn off when powering down (which seems normal when compared to non-bricked units that take a little while to power down after a freeze /  crash.  The 'test payload' also doesn't work (i.e. pressing A does not turn the unit off).  I ended up asking for feedback on the 3dshacks IRC channel, and after confirming more details about my apparantly bricked unit - such as the exact actions I carried out in relation to Plailect's guide and the exact contents on my SD card - I was advised that reporting this officially would be advisable as the only way to proceed  at this point would be hard mod recovery.

**Comment by james-d-elliott** [comment]
A picture of the glitched screen may help (not 100% sure).

**Comment by SG987** [comment]
The screens were black, with no graphical artifacts or actual glitching.

**Comment by AuroraWright** [comment]
This doesn't really make sense, as the installer always prints something to the screen, and never does anything (the code is pretty straightforward) without pressing SELECT. Any chances you pressed SELECT by mistake?

**Comment by SG987** [comment]
I'm pretty sure I didn't press select as the black screens implied to me that SafeA9LHinstaller failed to load. Had there been something I could class as a glitch I wouldn't have hesitated to press select (as per the only reference in the guide to troubleshooting failed SafeA9LHinstaller launching). 

In the case of an accidental pressing of select, is it more plausible that this tool (which I've never had an issue with before - thank you) bricked my unit?

 I realise that - like most issues with this process - user error is the likely culprit,l when something goes wrong, but I was *incredibly* diligent with the preparation, and would hate to waste everyone's time with bringing it up if I'd made a mistake somewhere.

**Comment by AuroraWright** [comment]
Yeah, I'd say if you pressed select while you were trying to force power off there's an high chance it powered off before the installation could complete.
